testfiles/HospitalBaseline.idf

```diff
@@ -57774,8 +57774,8 @@
     AHU2_OAMinOAFracSchedule,!- Minimum Fraction of Outdoor Air Schedule Name
     ,                        !- Maximum Fraction of Outdoor Air Schedule Name
     ,                        !- Mechanical Ventilation Controller Name
-    No,                      !- Time of Day Economizer Control Schedule Name
-    ,                        !- High Humidity Control
+    ,                        !- Time of Day Economizer Control Schedule Name
+    No,                      !- High Humidity Control
```

Comment:
@mjwitte All of these switches intentional?  From how far I've gotten into the PR description and the code changes I don't see the exact reason nor transition changes if we need to apply this change to user files.